website/docs/usage/v3-3.md

```diff
@@ -0,0 +1,222 @@
+
+<figure>
+
+** Intel Xeon W-2265 **
+
+| Model                                            | Avg. Words/Doc | v3.2 (WPS) | v3.3 (WPS) |   Diff |
+| :----------------------------------------------- | -------------: | ---------: | ---------: | -----: |
+| [`en_core_web_md`](/models/en#en_core_web_md)    |            100 |      17292 |      17441 |  0.86% |
+| (=same components)                               |           1000 |      15408 |      16024 |  4.00% |
+|                                                  |          10000 |      12798 |      15346 | 19.91% |
+| [`de_core_news_md`](/models/de/#de_core_news_md) |            100 |      20221 |      19321 | -4.45% |
+| (+v3.3 trainable lemmatizer)                     |           1000 |      17480 |      17345 | -0.77% |
+|                                                  |          10000 |      14513 |      17036 | 17.38% |
+
+</figure>
+
+<figure>
+
+**Apple M1**
+
+| Model                                            | Avg. Words/Doc | v3.2 (WPS) | v3.3 (WPS) |   Diff |
+| ------------------------------------------------ | -------------: | ---------: | ---------: | -----: |
+| [`en_core_web_md`](/models/en#en_core_web_md)    |            100 |      18272 |      18408 |  0.74% |
+| (=same components)                               |           1000 |      18794 |      19248 |  2.42% |
+|                                                  |          10000 |      15144 |      17513 | 15.64% |
+| [`de_core_news_md`](/models/de/#de_core_news_md) |            100 |      19227 |      19591 |  1.89% |
+| (+v3.3 trainable lemmatizer)                     |           1000 |      20047 |      20628 |  2.90% |
+|                                                  |          10000 |      15921 |      18546 | 16.49% |
+
+</figure>
+
+### Trainable lemmatizer {#trainable-lemmatizer}
+
+The new [trainable lemmatizer](/api/edittreelemmatizer) component uses
+[edit trees](https://explosion.ai/blog/edit-tree-lemmatizer) to transform tokens
+into lemmas. Try out the trainable lemmatizer with the
+[training quickstart](/usage/training#quickstart)!
+
+### displaCy support for overlapping spans and arcs {#displacy}
+
+displaCy now supports overlapping spans with a new
+[`span`](/usage/visualizers#span) style and multiple arcs with different labels
+between the same tokens for [`dep`](/usage/visualizers#dep) visualizations.
+
+Overlapping spans can be visualized for any spans key in `doc.spans`:
+
+```python
+import spacy
+from spacy import displacy
+from spacy.tokens import Span
+
+nlp = spacy.blank("en")
+text = "Welcome to the Bank of China."
+doc = nlp(text)
+doc.spans["custom"] = [Span(doc, 3, 6, "ORG"), Span(doc, 5, 6, "GPE")]
+displacy.serve(doc, style="span", options={"spans_key": "custom"})
+```
+
+import DisplacySpanHtml from 'images/displacy-span.html'
+
+<Iframe title="displaCy visualizer for entities" html={DisplacySpanHtml} height={180} />
```

Comment:
```suggestion
<Iframe title="displaCy visualizer for overlapping spans" html={DisplacySpanHtml} height={180} />
```

Reply:
Already done internally when working on the blog post, but I'll fix it in the linked main docs, too.